scripts/Logistics-Pipes.zs

```diff
@@ -843,8 +848,45 @@ recipes.addShaped(DisconnectWest, [
 [<ore:craftingToolScrewdriver>, DisconnectUp, null],
 [null, null, null]]);
 
+
 // ||||| Fluid Pipes |||||
-//Disabled for right now!
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 144, 1600, 120);

```

Comment:
GT++ 8k cell is an HV recipe. I'm not really sure if we should tier it that high. BTW this thing is the internal placeholder item for fluid transport. It has a max stack size of 1 and thus isn't very easy to handle. I suggest we replace all usage of this item with a) large steel cell b) 1k volumetric flask c) a new item we would invent

Reply:
Ok, i will cahnge it to an empty steel cell